templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,14 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production:
```

Comment:
Actually, I wonder why we need to set `url` to `DATABASE_URL` at all. I thought Heroku already did that? Can we drop this line and only set `pool`?